source/tutorials/fmt-cdiff-khanna.rst

```diff
@@ -506,6 +587,58 @@ We're also often interested in performing a differential abundance test at a spe
 .. question::
    Which genera differ in abundance across Subject? In which subject is each genus more abundant?
 
+
+Using q2-sample-classifier to predict sample characteristics
+------------------------------------------------------------
+
+Looks like CDI leads to a state of microbial dysbiosis. Could we use features of that dysbiosis state to diagnose CDI? Let's use :doc:`q2-sample-classifier <sample-classifier>` to find out. We will train a Random Forest classifier on a subset of our samples, and test the predictive accuracy of this model using another subset of our samples to determine how well this model generalizes to unseen samples. We will train our model to distinguish between pre- and post-FMT samples as a proxy measurement of current CDI:
+
+.. command-block::
+
+  qiime sample-classifier classify-samples \
+    --i-table disease-table.qza \
+    --m-metadata-file sample_metadata.tsv \
+    --m-metadata-column day_since_fmt \
+    --p-n-estimators 200 \
+    --p-random-state 666 \
+    --output-dir sample-classifier
+
+
+This action outputs a bunch of artifacts and visualizations, but we will focus on ``accuracy_results.qzv`` for now; this visualization shows us how frequently our trained classifier predicted CDI (or more precisely, how well it could distinguish sample pre- and post-FMT). We see an overall accuracy rate of 86.6%, much better than the error rate achieved by random guessing (53.3%). So this method is useful for differentiating pre- and post-FMT samples (and identifying differential features, as we will test below), but probably still not good enough to be considered a useful diagnostic tool!
+
+Random Forest models are very useful for feature selection, identifying features that are differentially abundant between sample classes. Features are assigned "importance" scores indicating their relative predictive value in the trained model. Let's check out the features that are most predictive of FMT status; we will make a heatmap of the abundance of the top 50 most important features, and tabulate their taxonomy:
+
+.. command-block::
+
+  qiime sample-classifier heatmap \
+     --i-table disease-table.qza \
+     --i-importance sample-classifier/feature_importance.qza \
+     --m-metadata-file sample_metadata.tsv \
+     --m-metadata-column day_since_fmt \
+     --p-cluster both \
+     --o-heatmap sample-classifier/heatmap.qzv \
+     --o-filtered-table sample-classifier/filtered-table.qza
+
+  qiime metadata tabulate \
+     --m-input-file sample-classifier/feature_importance.qza \
+     --m-input-file taxonomy.qza \
+     --o-visualization sample-classifier/top-features.qzv
+
+
+Using these actions, we see that:

```

Comment:
Perhaps worth adding more text on how did we arrive to these conclusions?

Reply:
I have added some more text on these specific interpretations